doc/source/train/examples/pytorch/distributing-pytorch/README.md

```diff
@@ -0,0 +1,442 @@
+    global_batch_size = 512
+
+    train_config = {
+        "lr": 1e-3,
+        "epochs": 1,
+        "batch_size_per_worker": global_batch_size // num_workers,
+    }
+
+    # [1] Start distributed training
+    # Define computation resources for workers
+    # Run `train_func_per_worker` on those workers
+    # ============================================
+    scaling_config = ScalingConfig(num_workers=num_workers, use_gpu=use_gpu)
+    run_config = RunConfig(storage_path="/mnt/cluster_storage", name="ray_train_run")
+    trainer = TorchTrainer(
+        train_loop_per_worker=train_func_per_worker,
+        train_loop_config=train_config,
+        scaling_config=scaling_config,
+        run_config=run_config,
+    )
+    result = trainer.fit()
+    print(f"Training result: {result}")
+
+if __name__ == "__main__":
+    train_cifar_10(num_workers=8, use_gpu=True)
+```
+
+Because you ran training in a data parallel fashion this time, it should have taken under 1 minute while maintaining similar accuracy.
+
+Head over to the Ray Train dashboard to analyze your distributed training job. You can do so by clicking on "Ray Workloads" and then "Ray Train," which should show you a list of all the training runs you have kicked off.
+
+![Train Runs](images/train_runs.png)
+
+Clicking into the run brings you to an overview page that shows logs from the controller, which is the process that coordinates your entire Ray Train job, as well as information about your 8 training workers.
+
+![Train Run](images/train_run.png)
+
+Clicking into an individual worker brings you to a more detailed worker page. 
+
+![Train Worker](images/train_worker.png)
+
+If your worker is still alive, you can click the "cpu flame graph," "stack trace," or "memory profiling" links in the overall run page or the individual worker page. Clicking the "cpu flame graph" link profiles your run with py-spy for 5 seconds and show you a cpu flame graph. Clicking the "stack trace" link shows you the current stack trace of your job - this is useful for debugging hanging jobs. Finally, clicking "memory profiling" profiles your run with memray for 5 seconds and show you a memory flame graph.
+
+You can also click the "Metrics" tab on the navigation bar to view useful stats about your cluster, such as gpu utilization and metrics about ray [actors](https://docs.ray.io/en/latest/ray-core/actors.html) and [tasks](https://docs.ray.io/en/latest/ray-core/tasks.html).
+
+![Metrics Dashboard](images/metrics_dashboard.png)
+
+## Step 3: Speed up its data loading with Ray Data
+
+Let’s modify this example to load data with Ray Data instead of the native Torch DataLoader. With just a few modifications, you can offload data preprocessing to an independently scaling distributed Ray Data pipeline. See [here](https://docs.ray.io/en/latest/data/comparisons.html#how-does-ray-data-compare-to-other-solutions-for-ml-training-ingest) for a comparison between Ray Data and Torch data loading.
```

Comment:
"offload data preprocessing to an independently scaling distributed Ray Data pipeline" is a mouthful and difficult for users to conceptualize what that actually means - we should revisit this to simplify the wording.